cfgov/paying_for_college/models/disclosures.py

```diff
@@ -359,7 +394,8 @@ def get_cohort(self, cohort):
         """
         cohorts = {
             'degrees_highest': School.objects.filter(
-                degrees_highest=self.degrees_highest),
+                degrees_highest=self.degrees_highest).exclude(
+                pk=FAKE_SCHOOL_PK),
```

Comment:
Why is the fake school excluded from the degrees cohort, but not the state or control one? Similarly, should these also filter on `operating=True`?

This makes me wonder if it would be worth creating a custom manager method to filter out this fake school, so this doesn't need to be repeated everywhere:

```py
class SchoolQuerySet(models.QuerySet):
    def operating(self):
        return self.filter(operating=True).exclude(pk=FAKE_SCHOOL_PK)

class School(models.Model):
    ...

    objects = SchoolQuerySet.as_manager()
```

This consolidates all logic around the "fake school" in one place, instead of having to worry about it everywhere. Then, anywhere you do anything with schools, you could use `School.objects.operating()....` to start the query and be guaranteed to have all valid schools.

(I realize I just suggested a very similar thing on https://github.com/cfpb/cfgov-refresh/pull/5367#discussion_r349640239, so maybe I'm just trying out my shiny new hammer.)

Reply:
Thank you for the catches on cohort exclusions.

The fake-school exclusion is tricky, because we need the fake school to show up in search and API calls, so that it can be used in demos and displayed in the tool, but we don't want it to throw off any cohort calculations, slight as the effects might be. 

It's a pretty obvious fake: Example University from Example Town, TX, url: example.com 
It does have a kicking mascot: the Fighting Exemplars!

It's also likely to be short-lived, as we hope not to need it after a demo period. So I'm not sure we want to embed it too deeply in code.

Reply:
Oh, my. The models don't need any cohort-calculation code. That's all handled now in the process_cohorts script. Removing (and removing the fake-school issue there).